modules/System Admin/alarm.php

```diff
@@ -40,34 +41,36 @@
     $fileUploader = new FileUploader($pdo, $gibbon->session);
     $fileUploader->getFileExtensions('Audio');
 
-    // Alram Types
+    // Alarm Types
     $alarmTypes = array(
         'None'     => __('None'),
         'General'  => __('General'),
         'Lockdown' => __('Lockdown'),
         'Custom'   => __('Custom'),
     );
 
-    $form = Form::create('alarmSettings', $_SESSION[$guid]['absoluteURL'].'/modules/'.$_SESSION[$guid]['module'].'/alarmProcess.php');
-
-    $form->addHiddenValue('address', $_SESSION[$guid]['address']);
-
-    $setting = getSettingByScope($connection2, 'System Admin', 'customAlarmSound', true);
+    $form = Form::create('alarmSettings', $gibbon->session->get('absoluteURL').'/modules/'.$gibbon->session->get('module').'/alarmProcess.php');
+    
+    $settingGateway = $container->get(SettingGateway::class);
+    
+    $settingAlarmSound = $settingGateway->selectBy(['scope' => 'System Admin', 'name' => 'customAlarmSound'])->fetch();
```

Comment:
In this case the SettingGateway has a helper method since getSettingByScope is so common :)
```suggestion
    $settingAlarmSound = $settingGateway->getSettingByScope('System Admin', 'customAlarmSound']);
```